zerver/decorator.py

```diff
@@ -559,7 +564,7 @@ def require_server_admin(view_func: ViewFuncT) -> ViewFuncT:
     @wraps(view_func)
     def _wrapped_view_func(request: HttpRequest, *args: object, **kwargs: object) -> HttpResponse:
         if not request.user.is_staff:
-            return HttpResponseRedirect(settings.HOME_NOT_LOGGED_IN)
+            return HttpResponseRedirect(DEFAULT_LOGIN_URL)
```

Comment:
I now feel that this change is kind of out of place. It should probably remain the same but with an assertion added.